test/app-tap/gh-2717-no-quit-sigint.test.lua

```diff
@@ -18,7 +18,7 @@ local prompt = 'tarantool> '
 local TARANTOOL_PATH = arg[-1]
 local test = tap.test('gh-2717-no-quit-sigint')
 
-test:plan(6)
+test:plan(8)
```

Comment:
When I run the test many times in parallel, it constantly fails.

The command:

```shell
$ ./test/test-run.py $(yes gh-2717 | head -n 1000)
```

The error:

```
[009] Last 15 lines of Tarantool Log file [Instance "app_server"][/tmp/t/009_app-tap/gh-2717-no-quit-sigint.test.lua.tarantool.log]:
[009] LuajitError: ...-meta/r/t-3/test/app-tap/gh-2717-no-quit-sigint.test.lua:194: time_quota is violated
[009] fatal error, exiting the event loop
```

I observed it on `master` too, so it is not related to your patch. Just highligthed it for you (I will resolve this discussion).